data/src/main/java/com/linku/data/implementation/repository/UserRepositoryImpl.kt

```
@@ -92,102 +47,46 @@ class UserRepositoryImpl @Inject constructor(
-        return true
     }
 
-    // BaseResponse<withDrawalResultDTO> 반환 → withAuth
     override suspend fun deleteUser(reason: String): Boolean {
-        val dto = DeleteUserRequestDTO(reason)
-        serverApi.withAuth(authPreference) { deleteUser(dto) }
-        return true
+        return try {
+            serverApi.deleteUser(DeleteUserRequestDTO(reason))
+            true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)